test/core/transport/chttp2/flow_control_test.cc

```diff
@@ -70,13 +73,17 @@ class TransportTargetWindowSizeMocker
   }
 
   void Reset() {
+    // Protecting access to variable window_size_ shared between client and
+    // server.
+    absl::MutexLock lock(&mu_);
     alternating_initial_window_sizes_ = false;
     window_size_ = kLargeInitialWindowSize;
   }
 
  private:
   bool alternating_initial_window_sizes_ = false;
```

Comment:
Can you add ABSL_GUARDED_BY annotations to these member variables?

Reply:
+1